labrad/types.py

```diff
@@ -447,16 +453,13 @@ def reprLRData(s):
 
 # LabRAD type classes
 
-class LRType(object):
+class LRType(with_metaclass(RegisterParser, object)):
```

Comment:
Interesting -- the `six` package also contains this shim. Is there a difference between `six`'s implementation and `future`'s?